modules/mbedtls/Kconfig.tls-generic

```diff
@@ -9,6 +9,8 @@ menu "TLS configuration"
 
 menu "Supported TLS version"
 
+if !(NRF_SECURITY || NORDIC_SECURITY_BACKEND)
```

Comment:
I thought one of these selected the other meaning we only need to be conditional on one of them?

Reply:
No. They can be either/or